python/Dockerfile

```diff
@@ -100,9 +100,9 @@ RUN find $PYTHON_INSTALL_LOCATION/bin -type f -exec sed -i "1s|^#!/usr/local/bin
 # Ensure pyenv works and it's the python version we expect
 RUN PYENV_VERSION=$PY_3_11 pyenv exec python --version | grep "Python $PY_3_11" || exit 1
 RUN bash /opt/python/helpers/build $PY_3_11
+# This is the default Python, so no need to tar it
 
-RUN pyenv global $PY_3_11
-
+FROM python-core
 # Install C-libs needed to build users' Python packages. Please document why each package is needed.
 USER root
 RUN apt-get update \
```

Comment:
This is the list of package installs that we now do concurrently with the python builds... It shouldn't take _that_ long, but nevertheless it's faster to do this concurrently, plus I expect this list to grow over time as users report C-libs that they need installed to build their deps.